### PR TITLE
Update for Github Issues Migration

### PR DESCRIPTION
**Pull request by denil-ct**
## Summary

- Added issue templates for feature request and bug report. (based on swift-docc)
- Updated feature requests and bug report sections of readme with the new templates.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests
- [ ] Ran the `./bin/test` script and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by denil-ct** [comment]
@ethan-kusters could you please review?

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: Bug Report
+about: Something isn't working as expected
+labels: bug
+---
+
+<!--
+	Thank you for contributing to Swift-DocC plugin!
```
Comment (ethan-kusters) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:8]:
```suggestion
	Thank you for contributing to the Swift-DocC plugin!
```

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+name: Feature Request
+about: A suggestion for a new feature
+labels: enhancement
+---
+
+<!--
+	Thank you for contributing to Swift-DocC plugin!
```
Comment (ethan-kusters) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:8]:
```suggestion
	Thank you for contributing to the Swift-DocC plugin!
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# of beginning with a blank one.
+#
+# For details, see: https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: false
+
+contact_links:
+  - name: Discussion Forum
+    url: https://forums.swift.org/c/development/swift-docc
+    about: Ask and answer questions about Swift-DocC
+  - name: Swift-DocC plugin Documentation
```
Comment (ethan-kusters) [inline .github/ISSUE_TEMPLATE/config.yml:19]:
```suggestion
  - name: Documentation
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    url: https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/
+    about: Read Swift-DocC Plugin's API documentation
```
Comment (ethan-kusters) [inline .github/ISSUE_TEMPLATE/config.yml:21]:
```suggestion
    about: Read about how to use the Swift-DocC Plugin in your projects
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+contact_links:
+  - name: Discussion Forum
+    url: https://forums.swift.org/c/development/swift-docc
+    about: Ask and answer questions about Swift-DocC
+  - name: Swift-DocC plugin Documentation
+    url: https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/
+    about: Read Swift-DocC Plugin's API documentation  
+  - name: Swift-DocC Documentation
+    url: https://www.swift.org/documentation/docc/
+    about: Learn about how to use the Swift-DocC documentation tool in your projects
```
Comment (ethan-kusters) [inline .github/ISSUE_TEMPLATE/config.yml:24]: The plugin's documentation site already links to Swift-DocC's documentation so I think we can leave this out for simplicity.

```suggestion
```

File: README.md [hunk 6]
```diff
@@ -86,10 +86,23 @@ For details on how to best build documentation for hosting online and a specific
...
+
+### Submitting a Bug Report
+
+Swift-DocC plugin tracks all bug reports with 
+[GitHub Issues](https://github.com/apple/swift-docc-plugin/issues).
+When you submit a bug report we ask that you follow the
+[provided template](https://github.com/apple/swift-docc-plugin/issues/new?template=BUG_REPORT.md)
+and provide as many details as possible.
+
+If you can confirm that the bug occurs when using the latest commit of Swift-DocC plugin
```
Comment (ethan-kusters) [inline README.md:99]:
```suggestion
If you can confirm that the bug occurs when using the latest commit of the Swift-DocC plugin
```

File: README.md [hunk 7]
```diff
@@ -86,10 +86,23 @@ For details on how to best build documentation for hosting online and a specific
 tutorial for publishing to GitHub Pages, please see 
 [the plugin's documentation](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/).
 
+## Bug Reports and Feature Requests
+
+### Submitting a Bug Report
+
+Swift-DocC plugin tracks all bug reports with
```
Comment (ethan-kusters) [inline README.md:93]:
```suggestion
The Swift-DocC plugin tracks all bug reports with 
```

**Review by ethan-kusters — COMMENTED** [review]
Thank you so much for opening this! It looks great.

**Comment by denil-ct** [comment]
Changes done
Please test and merge, thanks.

**Review by ethan-kusters — APPROVED** [review]
Thank you!

**Comment by ethan-kusters** [comment]
@swift-ci please test

**Pull request merged**